conda/recipes/rapids_dask_dependency/meta.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+package:
+- name: rapids_dask_dependency
+  version: 23.10.00.0
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - dask >=2023.7.1
+    - dask-core >=2023.7.1
+    - distributed >=2023.7.1
+
+about:
+  home: https://rapids.ai/
+  license: Custom
+  license_file: conda/recipes/rapids_dask_dependency/LICENSE
```

Comment:
The Dask & Distributed licenses are BSD-3-Clause. So we could use the same here for simplicity

Suggesting this particularly if we are shipping this package in our releases (as users may catch this in license auditing tools)

Reply:
I'm not sure how licensing a metapackage of ours that just includes other packages (no ours). Should we use a BSD-3-Clause license or an Apache license like the rest of RAPIDS?